volontulo/templates/organizations/organization_offers.html

```diff
@@ -18,7 +18,12 @@
             {% if o.status == 'STAGED' %}
                 <a href="{% url 'offers_view' o.title|slugify o.id %}">Pomóż</a>
             {% else %}
-                <a href="{% url 'activate_offer' o.id %}">Aktywuj</a>
+                <form id="offer_ {{ o.id }}_activate" method="post" action="{% url 'offers_edit' o.title|slugify o.id %}" enctype="multipart/form-data">
```

Comment:
ad. 1) - we need to develop some kind of verification for all levels of access on server-side. We will aproach it in near future.